projects/libreoffice/build.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -eu
+
+#shuffle CXXFLAGS -stdlib=libc++ arg into CXX as well because we use
+#the CXX as the linker and need to pass -stdlib=libc++ to build
+export CXX="$CXX -stdlib=libc++"
+#similarly force the -fsanitize etc args in as well as pthread to get
+#things to link successfully during the build
+export LDFLAGS="$CFLAGS -lpthread"
+
+cd $OUT
+#build under $OUT cause its the only place with enough space to do so
```

Comment:
unfortunate that the supplied demo build constraints don't match the actual one :-(